remme/rpc_api/event/_handlers.py

```diff
@@ -134,14 +134,33 @@ class BlockEventHandler(BaseEventHandler):
     def hash_keys(cls):
         return ('id',)
 
-    def prepare_response(self, state, validated_data):
-        return {'id': state}
+    async def prepare_response(self, state, validated_data):
+        """
+        Prepare response.
+
+        To fetch block by number, consider blocks like elements in array. To get second, count from zero.
+        """
+        block_information_client = BlockInfoClient()
+        block_information = await block_information_client.get_block_info(block_num=int(state.get('block_num')) - 1)
+
+        block_identifier = state.get('block_id')
+
+        return {
+            'id': block_identifier,
+            'timestamp': block_information.timestamp,
+        }
 
     def parse_evt(self, evt):
-        try:
-            return next(filter(lambda el: el['key'] == 'block_id', evt['attributes']))['value']
-        except StopIteration:
-            pass
+        LOGGER.info(f'evt is {evt}')
+        parsed_attributes = {}
+
+        for attribute in evt.get('attributes'):
```

Comment:
If we do not have `attributes`?

Reply:
@DeV1doR,[ is this even impossible](https://github.com/Remmeauth/remme-core/pull/189#discussion_r246433407)?

Reply:
@dmytrostriletskyi so why `.get` than?)

Reply:
@DeV1doR, I got you. You think, I always should use `dict['key']` syntax if I know the key is 100% presented in dictionary and `get` if I couldn't have key. But I do not consider these two different ways as you, I consider it from style guide and convenience of reading the code.

If consider your way as proper, `get` always should require `default value` because `key` couldn't be in dictionary, but this argument is optional, so I can use it as just fetching without any other things. 

Have you got me?

Reply:
@dmytrostriletskyi `import this` and pep-8 is your style guide)  So use a default value than if the error could be possible occurred. But more pythonic will be done through EAFP.

Reply:
@DeV1doR, you didn't convince me, but I have refactored to your proposal in respect to feature delivering.